### ISSUE TITLE
Update CONTRIBUTING.md with a full walkthrough

### ISSUE THREAD

**Issue by leogout** [QuantStack/voila#303]
Hey, I followed the instructions from the CONTRIBUTING.md but the setup.py [is pointing to voila.js](https://github.com/QuantStack/voila/blob/c399d66a2fb74364c5d7336ed615b153c21a3452/setup.py#L84) which [does not exist](https://github.com/QuantStack/voila/tree/master/share/jupyter/voila/templates/default/static)... 

### Command line output:
```
ERROR: Complete output from command python setup.py egg_info:
    ERROR: running egg_info
    running css
    running jsdeps
    rebuilding js and css failed
    missing files: ['C:\\Users\\leogout\\projects\\python\\voila\\share\\jupyter\\voila\\templates\\default\\static\\voila.js']
    error: [WinError 2] The specified file could not be found
```
Am I missing something ?

**Comment by leogout** [comment]
Related to #192

**Comment by leogout** [comment]
Hello again, so I simply had to do
```
cd js
npm install
```
I'll add this to the docs.

**Comment by jtpio** [comment]
Fixed by #304

**Comment by cnydw** [comment]
Just found out that under user root, the prepare script in package.json is skipped silently, which fails the dev installation.

So for people whose user is root, you may need to explicitly run `npm run prepare` after `npm install`.
[ref](https://npm.community/t/npm-ci-not-running-prepare-for-git-dependencies-when-run-as-root-in-docker/4485/5)
